eml/api/api.js

```diff
@@ -39,7 +39,7 @@ export const getSectionByid = async (sectionId) => {
   }
 };
 
-export const getExerciseByid = async (exerciseId) => {
+export const getExerciseById = async (exerciseId) => {
```

Comment:
Dont think it is a good idea to change the name, unless you have changed all the places that uses it to the correct one

Reply:
It have already been changed in all the the places where it is used (one place)